### PR TITLE
Updated PorterConfig type to use verbosity instead of debug and debug…

### PR DESCRIPTION
**Pull request by sgettys**
…-plugins

Signed-off-by: Steven Gettys <s.gettys@f5.com>

### CODE REVIEW

**Comment by sgettys** [comment]
#112 
#113

**Comment by sgettys** [comment]
Closes #112 and #113

File: docs/content/quickstart/_index.md [hunk 1]
```diff
@@ -278,6 +278,28 @@ Let's walk through the second method in detail.
      Digest: sha256:22cdfad0756c9ce1a8f4694b0411440dfab99fa2e07125ff78efe555dd63d73e
     ```
 
+## Environment Configuration
```
Comment (carolynvs) [inline docs/content/quickstart/_index.md:281]: 🎉

**Review by carolynvs — CHANGES_REQUESTED** [review]

**Review by carolynvs — APPROVED** [review]
Looks great, thanks!

**Pull request merged**